app-text/master-pdf-editor/master-pdf-editor-5.1.30.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils desktop
+
+DESCRIPTION="Master PDF Editor is a complete solution for viewing and editing PDF files"
```

Comment:
you should not out a package name  to the `DESCRIPTION` I've already fixed this in your package like twice.